SerialPrograms/Source/PokemonSV/Programs/ItemPrinter/PokemonSV_ItemPrinterRNG.cpp

```diff
@@ -25,6 +25,8 @@
 #include "PokemonSV/Inference/Dialogs/PokemonSV_DialogDetector.h"
 #include "PokemonSV/Inference/Overworld/PokemonSV_OverworldDetector.h"
 //#include "PokemonSV/Programs/PokemonSV_GameEntry.h"
+#include "PokemonSV/Programs/Farming/PokemonSV_MaterialFarmer.h"

```

Comment:
Programs shouldn't depend on other programs. Why are moving all the tools into the MaterialFarmer program?

Put all the common stuff in MaterialFarmerTools. And have ItemPrinterRNG and MaterialFarmer depend on MaterialFarmerTools.

I don't understand what the errors are.

Reply:
`MaterialFarmer_Descriptor` isn't being shared. Only the stats are . Pull the stats out into the tools file and leave `MaterialFarmer_Descriptor` in the MaterialFarmer program.

Reply:
Ok. I understand now. I didn't understand what you meant when you said it the first time.
I now see that the Stats aren't tied to the Descriptor.
I'll delete the errant commit, rebase, and fix it.